### ISSUE TITLE
Is there an easy way to downgrade or switch php versions?

### ISSUE THREAD

**Issue by litwi1rm** [teddysun/lamp#200]
Installed LAMP with php 8, having some issues so until I can resolve would like to downgrade to php 7.

**Comment by teddysun** [comment]
Rename PHP Install location `/usr/local/php` and reinstall it with same option.
```
mv /usr/local/php /usr/local/php8
```
and
```
./lamp.sh
```
Select PHP 7.

If `/usr/local/apache` and `/usr/local/mysql` already exists, it will be skip the installation when reinstall LAMP.

**Comment by litwi1rm** [comment]
![image](https://user-images.githubusercontent.com/7134135/119418930-21696080-bcc7-11eb-9fd1-dc898f892595.png)

**Comment by teddysun** [comment]
The new system is not compatible with the old software.

**Comment by litwi1rm** [comment]
appreciate the quick responses, so I also tried as suggested on ubuntu18 and got this error when installing php7.3
![image](https://user-images.githubusercontent.com/7134135/119422440-13b7d900-bccf-11eb-8ea7-90775ca22b80.png)

**Comment by ducna96** [comment]
![image](https://user-images.githubusercontent.com/35019158/119428837-d9365680-bd37-11eb-8b68-d057c4bef66c.png)
i want to reinstall php,, i I have chosen  do not install apache, and do not install mysql, and then script stop, it did not allow reinstall option , help me !
thanks.

**Comment by teddysun** [comment]
@ducna96 
No, you still need to chosen install Apache and MySQL. and other options need to chosen too.
If  path `/usr/local/apache` and `/usr/local/mysql` already exists, there will be skip the installation when reinstall LAMP.

**Comment by teddysun** [comment]
> appreciate the quick responses, so I also tried as suggested on ubuntu18 and got this error when installing php7.3
> ![image](https://user-images.githubusercontent.com/7134135/119422440-13b7d900-bccf-11eb-8ea7-90775ca22b80.png)

try PHP 7.4
It looks like ICU problem

**Comment by litwi1rm** [comment]
getting the same issue with 7.4

I also spun up a new azure ubuntu 20 vm, and unable to successfully run the script with php5.6 or 7.0
